README.md

```diff
@@ -70,19 +70,23 @@ Set up a new local CMS instance.
 
 3. Build & Start the Docker Containers:
 
-    ```sh
-    make start
-    ```
+    | For Testing | For Developing & Testing |
+    | - | - |
+    | `make start` | `docker-compose -f ./docker-compose.dev.yml up` |
+
+    > **Note**
+    > This will make the terminal window busy. To run commands after this, **either** open a new terminal window **or** run `docker-compose -f ./docker-compose.dev.yml up --detach` instead.
 
 4. Enter the CMS Docker Container:
 
     (This opens a command prompt within the container.)
 
     ```sh
     docker exec -it core_cms /bin/bash
+    # This opens a command prompt within the container
     ```
 
-5. Run the Django Application:
+5. Update the Django Application:
```

Comment:
The application is already running via `make start` or `docker compose … up` above.